x-pack/plugins/cases/server/routes/api/internal/bulk_get_attachments.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import type { BulkGetCommentsRequest } from '../../../../common/api';
+
+import { INTERNAL_BULK_GET_ATTACHMENTS_URL } from '../../../../common/constants';
+import { createCaseError } from '../../../common/error';
+import { createCasesRoute } from '../create_cases_route';
+import { escapeHatch } from '../utils';
+
+export const bulkGetAttachmentsRoute = createCasesRoute({
+  method: 'post',
+  path: INTERNAL_BULK_GET_ATTACHMENTS_URL,
+  params: {
+    params: schema.object({
+      case_id: schema.string(),
```

Comment:
Not really a question but I thought in the end you had decided to have this API work for only one case.

This will only be used in the Case detail view right?